baselines/Google Drive and Docs Minimum Viable Secure Configuration Baseline v0.2.md

```diff
@@ -260,17 +261,17 @@ This section covers whether a security update issued by Google will be applied t
 ### Policies
 
 #### GWS.DRIVEDOCS.3.1v0.2
-Agencies SHALL enable security updates for Drive files.
+Agencies SHALL enable the security update for Drive files.
 
-- _Rationale:_ Google may add new security features over time. Allowing security updates helps ensure that your files are protected with the latest features Google makes available.
+- _Rationale:_ By not enabling the resource key security update it creates the potential for an unauthorized access to files. Enabling this security update mitigates the risk by ensuring access is controlled properly.
```

Comment:
The first sentence doesn't work grammatically and the second promises too much (access can still be improperly controlled even if this update is applied).

Reply:
By not enabling the update to the resource key security update a user could potentially gain unauthorized access to files. Enabling this security update decreases risk of unauthorized access and data spillage by controlling access to files in Google Drive. 

@bnewlin-MITRE @adhilto I didn't change much from the original but addressed grammatical issues and the over promising